conf/openshift/openshift.json

```diff
@@ -249,7 +250,7 @@
             "containers": [
               {
                 "name": "dataverse-plus-glassfish",
-                "image": "iqss/dataverse-glassfish:latest",
+                "image": "tmonson/dataverse-glassfish:latest",
```

Comment:
Can't commit this change.  Need to undo it for the PR.

Reply:
Fixed.